### PR TITLE
SEARCH-1781 : Add initial tests for elasticsearch layer.

### PR DESCRIPTION
**Pull request by uppinder**
Contains tests for checking whether elasticsearch and kibana are fully functional.

### CODE REVIEW

**Comment by lplazas** [comment]
Please change the pull request name to begin with your Jira ticket number so that it is picked up by automation.
E.g. SEARCH-1322 Add hello world

File: Tests/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install **requests**.
+
+```bash
+pip install requests
+```
+
+## Executing tests
+
+Make sure a cluster is up and running and the terminal session is connected to the EKS cluster.
```
Comment (lplazas) [inline Tests/README.md]: You could reference the confluence documentation link on how to connect to the EKS cluster.

File: Tests/test_elasticsearch.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+import requests
+
+
+def init(cluster_name):
+    # Initialize logging module
+    logging.basicConfig(level=logging.INFO, format='%(asctime)s - [%(levelname)s] - %(message)s', datefmt='%Y-%m-%d %H:%M:%S')
+    logging.info("The cluster name is: " + cluster_name)
+
+    # Record Elasticsearch and Kibana Endpoint
+    ELB_ENDPOINT    = 'https://' + os.popen('kubectl get svc -o json | jq \'.items[] | select(.metadata.name == "' + cluster_name + '-es-http") | .status.loadBalancer.ingress[0].hostname\'').read().rstrip().strip('"') + ':9200'
```
Comment (lplazas) [inline Tests/test_elasticsearch.py]: Not too important, but you can easily select one svc by running:
`kubectl get svc/cluster_name-es-http`

File: Tests/test_elasticsearch.py [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+import os
+import json
+import yaml
+import logging
+import argparse
+import requests
+
+
+def init(cluster_name):
```
Comment (lplazas) [inline Tests/test_elasticsearch.py]: SuggestionL Before getting the ELBs endpoints, we should check that they exist. If they don't, the test should quit.

Comment (uppinder) [reply Tests/test_elasticsearch.py]: I was thinking of adding a another series of tests to make sure the K8s infra is up. The current PR only addresses the elasticsearch testing of things.

File: Tests/test_elasticsearch.py (hunk 2 above)
Comment (uppinder) [reply Tests/test_elasticsearch.py]: Thanks, I overlooked this.

File: Tests/test_elasticsearch.py (hunk 3 above)
Comment (lplazas) [reply Tests/test_elasticsearch.py]: The separate tests for the K8S infra sound good, but I still think that we should check that either the ELB endpoints exist or that there is a service running on that DNS:PORT before running everything else. 
As the tests expand and get more complex, they will start taking more time, but if there's nothing running, they will all always fail.

Comment (uppinder) [reply Tests/test_elasticsearch.py]: Agreed on the fail all tests part. Will add a commit for it.

Comment (uppinder) [reply Tests/test_elasticsearch.py]: Commit added, lmk if that works.

File: Tests/test_elasticsearch.py (hunk 2 above)
Comment (uppinder) [reply Tests/test_elasticsearch.py]: Commit added for the same.

**Pull request merged**